tests/test_create_order.py

```diff
@@ -7,6 +8,7 @@
 
 class TestCreateOrder:
 
+    @pytest.mark.run(order=1)
```

Comment:
could you use a `pytest.fixture` for this instead? Could also use that to clean up the order after as well if that makes sense.

Reply:
Updated to use fixture instead. I don't see an endpoint to delete an order so how would we go about cleaning the order up after?